docs/benchmarks/media-streaming.md

```diff
@@ -28,21 +28,22 @@ To set up the dataset you have to first `pull` the dataset image and then run it
 
 The following command will run the dataset container that exposes the video dataset volume, which will be used by the streaming server:
 
-    $ docker run --name streaming_dataset cloudsuite/media-streaming:dataset
+    $ docker run --name streaming_dataset cloudsuite/media-streaming:dataset ${DATASET_SIZE} ${SESSIONS_COUNT}
 
 Copy logs folder from container to Host1:
 
     $ docker cp ${DATASET_CONTAINER_ID}:/videos/logs $HOME
     
+`DATASET_SIZE`, in GBs, scales the size of the dataset to the given number. If you don't specify `DATASET_SIZE`, by default, the dataset container generates a collection of 10 videos for each of 240p, 360p, 480p, and 720p resolutions, having around 3.5 GB size altogether. For each of the resolutions, the dataset container creates a log file containing the sessions. Each session represents the behavior of a specific client streaming a specific media by mentioning the name of the requested media and the bytes streamed in each connection. These sessions will be read and processed by the client container. `SESSIONS_COUNT` specifies the number of sessions for each resolution with 5 as the default value. It would be good to give an appropriate value to `SESSIONS_COUNT` to make sure all videos in the dataset would be accessed. Note that the dataset container uses a specific distribution to simulate the behvaior that some videos are more popular than the others. Check the logs of the dataset container to see statistics regarding the generated sessions for various resolutions.
```

Comment:
We need to improve this explanation:
- Add a general description of dataset access log session generator. How it works, and how the clients use it.
- Also be clear on the fact these session are not the dynamic number that the client uses, these are just all videos that can possibly be accessed. The client loops over this "session list" to access certain videos.